docs/data/joy/components/toggle-button/toggle-button.md

```diff
@@ -0,0 +1,53 @@
+---
+product: joy-ui
+title: React Toggle Button component
+githubLabel: 'component: toggle button'
+---
+
+# Toggle Button
+
+<p class="description">A Toggle Button can be used to group related options.</p>
+
+:::info
+💡 The Joy UI Toggle Button component is still in development.
+If you're in need of it, please upvote [**this issue**](https://github.com/mui/material-ui/issues/36617) to help us prioritize the next batch of new components.
+:::
+
+## Integration with headless UI libraries
+
+In the meantime, you can still adopt Joy UI today for building a toggle button!
+
+This document shows how to construct it with existing Joy UI components combined with popular headless UI libraries.
+
+### Using the `IconButton` component
+
+Joy UI's [`IconButton`](/joy-ui/react-button/#icon-button) component is perfect for building a toggle button because it already produces a square button with an icon inside.
```

Comment:
```suggestion
Joy UI's [Icon Button](/joy-ui/react-button/#icon-button) component is perfect for building a toggle button because it already produces a square button with an icon inside.
```